spec/acceptance/foreman_basic_spec.rb

```diff
@@ -33,10 +33,10 @@ class { '::apache::mod::passenger':
 
     # Actual test
     class { '::foreman':
-      repo                => 'nightly',
-      user_groups         => [],
-      admin_username      => 'admin',
-      admin_password      => 'changeme',
+      repo                        => 'nightly',
+      user_groups                 => [],
+      initial_admin_username      => 'admin',
```

Comment:
A bit odd alignment here and in some other files. The other acceptance tests also have a bit more whitespace than I'd expect